packages/arro3-core/test_arro3_core.py

```diff
@@ -0,0 +1,8 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["pyarrow", "numpy", "pandas"])
```

Comment:
Don't you want `packages=["arro3"]` here? You don't directly import any of the others, so they shouldn't be in the list here.

Reply:
oh yes, thank you. I copied and pasted from the pyarrow recipe